### PR TITLE
Add basic inbox list UI

### PR DESCRIPTION
**Pull request by ywsang**
Implement the container for the Inbox section and start the UI for the normal state of the Inbox list items. 
image

**IM SORRY MY BRANCH NAME IS STUPID I WASN'T PLANNING ON WORKING ON THE BRANCH BUT THEN I DID**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
...
-  extends: ['airbnb-typescript'],
-  parserOptions: {
-    project: './tsconfig.json',
-  },
-  rules: {
-    // Get rid of these two once we wanna deal w accessibility stuff
-    'jsx-a11y/interactive-supports-focus': 0,
-    'jsx-a11y/click-events-have-key-events': 0
-  }
-};
```
Comment (Omarrasheed) [inline .eslintrc.js]: Ummm lets not delete this plz

File: src/components/HomePage/styles.scss [hunk 2]
```diff
@@ -2,4 +2,6 @@
   display: flex;
   flex-direction: row;
   align-items: stretch;
+  height: 100%;
+  background-color: $primaryBlue;
```
Comment (Omarrasheed) [inline src/components/HomePage/styles.scss]: jw whats this

File: src/components/SideBar/index.tsx [hunk 3]
```diff
@@ -25,7 +25,7 @@ class SideBar extends React.Component<SideBarProps> {
     return (
       <div className="background">
```
Comment (Omarrasheed) [inline src/components/SideBar/index.tsx]: I think u can remove this if u changed the name to `container` which seems to be like some sort of html keyword

File: src/components/Inbox/index.tsx [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import { Checkbox, Divider, IconButton } from '@material-ui/core';
+import { StylesProvider } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+import './styles.scss';
+
+const Inbox: React.FunctionComponent = () => (
+  <StylesProvider injectFirst>
```
Comment (Omarrasheed) [inline src/components/Inbox/index.tsx]: j curious what does this do

File: src/components/Inbox/index.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+          <ListItemText
+            classes={{
+              root: 'text',
+              primary: 'primary-text',
+              secondary: 'secondary-text',
+            }}
+            primary="Button not working"
+            secondary="The popular times button does nothing"
+          />
+        </ListItem>
```
Comment (Omarrasheed) [inline src/components/Inbox/index.tsx]: Can we potentially move this to a function to be used with like a `forEach` or something? I have no clue how that'll work but i'd imagine we're gonna need to do this at some point. Also if u wanna keep that for another PR feel free but just wanted to bring it up

Comment (ywsang) [reply src/components/Inbox/index.tsx]: yea saw some examples online for using a loop in the render to generate the items based on the list that i'll add in a later pr -- was just doing the ui for the cells in this PR

File: src/components/Inbox/index.tsx (hunk 4 above)
Comment (ywsang) [reply src/components/Inbox/index.tsx]: something along the lines of prioritize the custom css rules over the material ui native css rules

File: src/components/HomePage/styles.scss (hunk 2 above)
Comment (ywsang) [reply src/components/HomePage/styles.scss]: me exploring for real to see what was what
i'll remove

**Pull request merged**